sycl/include/CL/sycl/properties/function_pointer.hpp

```diff
@@ -0,0 +1,93 @@
+//==----------- function_pointer.hpp --- SYCL Function pointers ------------==//
```

Comment:
This seems like an extra copy, the same comment applies to a few more files in `sycl/include/CL/sycl/properties` directory

Reply:
Good catch! Removed